source/extensions/filters/udp/udp_proxy/udp_proxy_filter.h

```diff
@@ -236,6 +236,7 @@ class UdpProxyFilter : public Network::UdpListenerReadFilter,
     // packets from the upstream host. Note that a a local ephemeral port is bound on the first
     // write to the upstream host.
     const Network::SocketPtr socket_;
+    bool skip_connecting_{};
```

Comment:
nit: Please add a comment which explains why this would be true vs false and what it controls (if true, does not call `connect()` on the socket).

Also, if it skips `connect()` should it be called `skip_connect_`

Reply:
Fixed, thanks.